apollo-http-cache/src/main/java/com/apollographql/apollo/cache/http/ResponseHeaderRecord.java

```diff
@@ -272,7 +273,7 @@ private void writeCertList(BufferedSink sink, List<Certificate> certificates)
   Response response() {
     RequestBody body = null;
     if (HttpMethod.permitsRequestBody(requestMethod)) {
-      body = RequestBody.create(MediaType.parse("application/json; charset=utf-8"), "");
+      body = RequestBody.create(MediaType.parse(ApolloMediaType.APPLICATION_JSON_UTF8), "");
```

Comment:
Any reason to keep the charset here?

Reply:
Initial thought was that it won't affect the content-type header, but realized OkHttp uses the content type set in the body. Have removed the charset. Thanks.